connect/runtime/src/test/java/org/apache/kafka/connect/integration/ExactlyOnceSourceIntegrationTest.java

```diff
@@ -266,6 +267,7 @@ public void testPollBoundary() throws Exception {
         props.put(NAME_CONFIG, CONNECTOR_NAME);
         props.put(TRANSACTION_BOUNDARY_CONFIG, POLL.toString());
         props.put(MESSAGES_PER_POLL_CONFIG, Integer.toString(recordsProduced));
+        props.put(THROUGHPUT_CONFIG, Integer.toString(recordsProduced));
```

Comment:
the config is a Long, so these settings could be 
Long.toString(100L)
I checked the test that OOM'ing for me too and the number of records actually produced with your setting is still much larger than actually erquired.
I found using the same variable `recordsProduced` for throughput was a bit puzzling, maybe just using another literal would be ok.

Reply:
even 50 msgs/sec will be enough

Reply:
Since the result of Integer.toString(100) and Long.toString(100L) are the same, I don't think this necessary.
The reason I re-used the same variable was because I wanted to keep the runtime of the test constant. If there were two variables, someone could tune one while holding the other constant until the test timed out.

I agree that `recordsProduced` is a poor name, because this test produces many more records than that under normal conditions. Do you have a better name in mind?

Reply:
Again, it may not be *necessary* to use Long instead of Integer but it helps.
And using two variables instead of one, although with related values is again helping readability.
Reusing the same one is something that makes me stop and think "why...?"
So having a 2nd variable e.g. like
`long throughput_msgs_sec = recordsProduced / 2L;`
would be my preference (and a short line comment for it) e.g.
// need to limit actual records.count() to avoid OOM